### DISCUSSION TITLE
GCSToBigQueryOperator Error: Bad character (ASCII 0) encountered.

### DISCUSSION THREAD

**Discussion by fairy793** [apache/airflow#24878] [General]
### Apache Airflow version

2.3.2 (latest released)

### What happened

when i load csv data from gcs to bigquery enconter this error  :'message': 'Error while reading data, error message: Error detected while parsing row starting at position: 21. Error: Bad character (ASCII 0) encountered.'}. what should i do ?

### Operating System

linux 

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Certainly you should not open a "bug" in Airflow. You had clear instructions that if you have question you should open discussion.

**Comment by potiuk** [comment]
BTW. You have bad characters in your file. you need to fix them most likely. But this is not an Airflow problem - it's the problem that your .csv file is badly formatted. And the onl person that can do anything about is you, so reaching out here is not the best idea.

**Reply by fairy793** [reply]
i upload the data from mysql to google cloud storage using MySQLToGCSOperator,how cloud i formated the data correctly ?
What do I need to do and how do I set the parameters?

**Reply by potiuk** [reply]
No idea. Consult your documentation.

**Reply by potiuk** [reply]
You really need to check your file and see what is wrong with it... Just analyse it and understand CSV format, make sure that both operators produce/expect the same format. You really need to learn thos things if you want to effectively debug it. This is your job.

**Reply by fairy793** [reply]
Thanks for your quick reply！I found that the file contains special characters like '<0x000'，delete these characters and there is no such error.
